operator/src/config.rs

```diff
@@ -55,11 +56,59 @@ impl Config {
                 .expect("METRICS_DELAY must be a number"),
         );
 
+        let query_timeout = match env::var("QUERY_TIMEOUT") {
+            Ok(val) => val.parse::<u64>().expect("QUERY_TIMEOUT must be a number"),
+            Err(_) => 120000,
```

Comment:
`rust

let statement_timeout = env::var("STATEMENT_TIMEOUT")
            .map(|e| e.parse::<u64>().expect("STATEMENT_TIMEOUT must be a number")).unwap_or(120000)
`

Reply:
No problem for me, I think the pattern matching is more clear for the use case but I'll change it.

Reply:
Mmm sory now that I think about it, doing it with `unwrap_or` is rather uncomfortable because of the types. If we do `unwrap_or` we will get a string if the value is set on the env, and we would like to have a u64 if the env is not set. We could do something like:

```rust
let statement_timeout = env::var("STATEMENT_TIMEOUT")
    .unwrap_or("120000")
    .parse::<u64>()
    .expect("STATEMENT_TIMEOUT must be a number");
```

but I find it weird to have the `"120000"` written there as a string. Still, I don't really have a problem using the unwrap. Let me know what you think.

Reply:
example, but only if you like. It's only a suggestion.
`let timetout = env::var("ENV")
.map(|e| e.parse::<u64>().expect("ENV must be a number"))
.unwrap_or(12000);`